### ISSUE TITLE
Request: Different custom css for mail, calendar and drive 

### ISSUE THREAD

**Issue by DFelten** [Thomas101/wmail#491]
It would be great if we could configure different custom css for mail, calendar and drive. So we can define different styles for each service. Thanks for the latest update, it's great to have a tool for Google Mail and the calendar!

* Which version of WMail are you using? [e.g. 1.3.1]
2.0.8

* Which Operating System are you using? [e.g. Windows 10, Mac OSX 10, Ubuntu 14.04]
Windows 10

**Comment by codeman6100** [comment]
I concur this would be useful.

**Comment by Thomas101** [comment]
Yeah I agree. Will add it in soon. In the short term you could check the url of the page, although you'd also then have to inject your css with js.

**Comment by DFelten** [comment]
There is a problem with injecting css with javascript. If I insert custom CSS for Inbox, it is inserted only on the respective page, but the sites share the same CSS. Removing the changes is very complex, especially with many changes.
